src/main/java/com/blibli/future/vo/ApprovalRequestVo.java

```diff
@@ -23,7 +25,7 @@ public void setNik(String nik) {
 		this.nik = nik;
 	}
 
-	public boolean isApproved() {
+	public boolean getIsApproved() {
```

Comment:
Well, I know that this is just a getter, but the name `getIsApproved` is kind of weird